conan/tools/cmake/cmake.py

```diff
@@ -98,12 +98,28 @@ def configure(self, variables=None, build_script_folder=None):
 
         arg_list.extend(['-D{}="{}"'.format(k, v) for k, v in self._cache_variables.items()])
         arg_list.append('"{}"'.format(cmakelist_folder))
+        arg_list.append(self._get_log_level_arg())
 
         command = " ".join(arg_list)
         self._conanfile.output.info("CMake command: %s" % command)
         with chdir(self, build_folder):
             self._conanfile.run(command)
 
+    @staticmethod
+    def _get_log_level_arg():
+        """
+        cmake --log-level=<ERROR|WARNING|NOTICE|STATUS|VERBOSE|DEBUG|TRACE>
```

Comment:
On a second thought, I think I would decouple, and the build system verbosity is different to the Conan CLI verbosity. The tools/build system verbosity, should be controlled independently from ``conf``, in any case it would be nice to have a ``conf`` like ``tools.build:verbosity`` that abstracts things a bit, but that is all. Lets leave this out of this PR?